packages/orchestration/src/withdraw-reward.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+
+// question: are we doing snake-case filenames?
+
+import { E } from '@endo/far';
+import {
+  MsgWithdrawDelegatorReward,
+  MsgWithdrawDelegatorRewardResponse,
+} from '@agoric/cosmic-proto/cosmos/distribution/v1beta1/tx.js';
+import { Any } from '@agoric/cosmic-proto/google/protobuf/any';
+import { decodeBase64 } from '@endo/base64';
+
+const { Fail } = assert;
+
+/**
+ * @import { ChainAccount } from './types.js';
+ * @import {AnyJson} from '@agoric/cosmic-proto';
+ */
+
+/** @param {{ delegatorAddress: string, validatorAddress: string}} fields */
+export const makeWithdrawDelegatorRewardProtoMsg = ({
+  delegatorAddress,
+  validatorAddress,
+}) =>
+  MsgWithdrawDelegatorReward.toProtoMsg({
+    delegatorAddress,
+    validatorAddress,
+  });
+
+// XXX why doesn't Any.toJSON return AnyJson???
+/**
+ * @param {Any} x
+ * @returns {AnyJson}
+ */
+const toJSON = x => /** @type {AnyJson} */ (Any.toJSON(x));
+
+// ???: issue / spec calls this WithdrawDelegatorReward
+// am I right to take that literally? don't most of our function
+// names start with lower-case?
```

Comment:
Where do you see this? I see `withdrawRewards` and `withdrawReward` on `OrchestrationAccountBase`. Maybe you are seeing `WithdrawRewards` for an `invitationMaker` somewhere?

Reply:
In the issue, https://github.com/Agoric/agoric-sdk/issues/9071 :
> The `WithdrawDelegatorReward` API should ...

Reply:
OrchestrationAccountBase? I don't see that anywhere.